Libraries/Utilities/MessageQueue.js

```diff
@@ -26,9 +26,11 @@ const METHOD_IDS = 1;
 const PARAMS = 2;
 const MIN_TIME_BETWEEN_FLUSHES_MS = 5;
 
-const TRACE_TAG_REACT_APPS = 1 << 17;
+const TO_NATIVE = 1;
+const TO_JS = 0;
+const defaultSpy = (info)=>console.log(`${info.type == TO_JS ? 'N->JS' : 'JS->N'} : ${info.module ? (info.module+'.') : ''}${info.method}(${JSON.stringify(info.args)})`);
```

Comment:
Can you split this over multiple lines? Since it's also only used in one spot, I'd just define it inline in `spy`